src/uct/api/uct.h

```diff
@@ -1205,7 +1205,8 @@ struct uct_listener_params {
 
     /**
      * Backlog of incoming connection requests.
-     * If not specified, SOMAXCONN, as defined in <sys/socket.h>, will be used.
+     * If not specified, each CM component will use its maximal allowed value,
+     * based on the system's setting.
```

Comment:
should we specify the backlog must be positive?

Reply:
/**
     * Backlog of incoming connection requests. If specified, must be a positive value.
     * If not specified, each CM component will use its maximal allowed value,
     * based on the system's setting.
     */

ok?

Reply:
@yosefe what do you think?

Reply:
ok